lib/Resource/Wallet.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MagicAdmin\Resource;
+
+abstract class Wallet
```

Comment:
Nit, probably better to use enums here. https://www.php.net/manual/en/language.types.enumerations.php

Reply:
I was considering this but enums are only supported in 8.1 - the SDK currently supports `PHP 5.6.0 and later` so I didn't want there to be a breaking change for existing clients for this small code change.

Reply:
ohhh interesting. Are we really that old here? This is fine then.

Reply:
Yup! Either we're super old or PHP just moves quick 😆